Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1272,6 +1272,20 @@ CSSRhythmicSizingEnabled:
     WebCore:
       default: false
 
+CSSScopingStylesEnabled:
+  type: bool
+  status: preview
+  category: css
+  humanReadableName: "CSS Scoping Styles: @scope rule"
```

Comment:
Can we name it either `CSSScopeAtRuleEnabled` or `CSSScopingEnabled` ?

also the humanReadableName should be "CSS Scoping (@scope)" or something like that.

Reply:
CSS Scoping is a bit overlapping with Shadow DOM. Let's use the more explicit `CSSScopeAtRule`